prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -285,4 +303,31 @@ public void verify() {
             }
         }
     }
+
+    private void printListOfProfiles() throws Exception {
```

Comment:
1. The output needs to list the name of the profile.
2. When multiple profiles are present there's no separation between them
3. Would be nice to structure it a bit more - maybe indent the channel and feature packs section so it's clear they're part of a profile.